From df010031b9641332fed0cb3349ded04290f539c3 Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <heinrich.schuchardt@canonical.com>
Date: Thu, 2 Sep 2021 13:37:52 +0200
Subject: [PATCH 1/1] shim: avoid BOOTx64.EFI in message on other architectures

An error message complaining about missing file BOOTx64.EFI makes no
sense on other architectures.

Signed-off-by: Heinrich Schuchardt <heinrich.schuchardt@canonical.com>
---
 shim.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/shim.c b/shim.c
index 94a5176..1e774f7 100644
--- a/shim.c
+++ b/shim.c
@@ -724,7 +724,7 @@ should_use_fallback(EFI_HANDLE image_handle)
 	efi_status = gBS->HandleProtocol(image_handle, &EFI_LOADED_IMAGE_GUID,
 					 (void **)&li);
 	if (EFI_ERROR(efi_status)) {
-		perror(L"Could not get image for bootx64.efi: %r\n",
+		perror(L"Could not get image for boot" EFI_ARCH L".efi: %r\n",
 		       efi_status);
 		return 0;
 	}
-- 
2.32.0

